utils/efushell/integrationtest.py

```diff
@@ -0,0 +1,115 @@
+import json
+import subprocess
+import time
+from EFUMetrics import Metrics
+
+
+def compare_pair(x, y, operator):
+    if operator == "==":
+        return x == y
+    if operator == ">=":
+        return x >= y
+    print("invalid operator")
+    return False
```

Comment:
maybe it is better to raise an exception here to be able to distinguish between the check and the valid command?